mediachain/reader/api.py

```diff
@@ -56,12 +56,24 @@ def with_fallback():
     return with_thumb
 
 
+def merge(a, b, path=None):
```

Comment:
@autoencoder I think you have a more comprehensive version of this that you used to merge schemas, right?